watertap/data/techno_economic/deep_well_injection.yaml

```diff
@@ -0,0 +1,33 @@
+default:
+  recovery_frac_mass_H2O:
+    value: 0.9999
+    units: dimensionless
+  default_removal_frac_mass_solute:
+    value: 0
+    units: dimensionless
+  capital_cost:
+    cost_factor: None
+    reference_state:
```

Comment:
I don't think this is used, so it can be removed.

Reply:
I used the inlet `flow_vol` for costing (sizing term). I added `basis: flow_vol` here to clarify.

Reply:
I was referring to `reference_state`, which is not used at all in the code so shouldn't be in the yaml file either (there is a separate flow basis reference later which covers this, but is also used elsewhere in the model).

Reply:
@luohezhiming @andrewlee94 I see that `reference_state` has been removed from this YAML file, but `flow_basis` is still included. Is it necessary to remove the latter before merging this PR?

Reply:
@tarnold17 In this case `flow_basis` is still required for the actual unit model (rather than just the capital cost calculations as in other cases). Good question to ask however.